File: java/Product.java
class Product{

    //atribut
    private String idProduct;
    private int price;

    Product(){
    }

    //setter dan getter
    public void setIdProduct(String idProduct){
        this.idProduct = idProduct;
    }

    public String getIdProduct(){
        return this.idProduct;
    }

    public void setPrice(int price){
        this.price = price;
    }

    public int getPrice(){
        return this.price;
    }
}

File: c++/product.cpp
#include <string>
#include <iostream>

class Product{

    //atribut
    private:
        string idProduct;
        int price;

    public:
        //constructor
        Product(){
            this->idProduct = "";
            this->price = 0;
        }

        //setter dan getter
        void setIdProduct(string idProduct){
            this->idProduct = idProduct;
        }

        string getIdProduct(){
            return this->idProduct;
        }

        void setPrice(int price){
            this->price = price;
        }

        int getPrice(){
            return this->price;
        }

        //destructor
        ~Product(){
        }
};